content/docs/setup-and-administration/getting-access-tokens-using-app-registrations/index.md

```diff
@@ -88,8 +90,8 @@ If you do not yet have an App Registration configured please follow [the first-t
 ```bash
 curl https://app.rkvst.io/archivist/iam/v1/appidp/token \
     --data-urlencode "grant_type=client_credentials" \
-    --data-urlencode "client_id=${CLIENT_ID}" \
-    --data-urlencode "client_secret=${SECRET}"
+    --data-urlencode "client_id=$CLIENT_ID" \
+    --data-urlencode "client_secret=$SECRET"
```

Comment:
adding the {} explicitly every time makes it clearer - reading this quickly confuses the '$' and the 'S' of $SECRET for example

Reply:
On a related note the section with the curl command has "@BEARER_TOKEN_FILE"

it is unobvious that the $ is missing where as

"@${BEARER_TOKEN_FILE}

makes it clearer if the $ is accidentally left out...

Reply:
I am going to leave this for now as I think this is a one off miss and I wouldn't expect it normally